app.py

```diff
@@ -19,9 +19,11 @@
 
 
 @app.route("/")
-def hello():
-    return "In the words of Théoden....<br>'So... it beings...'"
-
+def index():
+    organisations = json.load(open('demo_data.json'))
```

Comment:
Could you please revert this change? You should be able to insert a few organisations in the mongo database and load them on the frontend the exact same way you are doing right now as well. If you need help with this please let me know.